deploy/operator/operator.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nmstate-operator
+  namespace: {{ .OperatorNamespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: nmstate-operator
+  template:
+    metadata:
+      labels:
+        app: kubernetes-nmstate
```

Comment:
let's have two `app` labels for knmstate

```yaml
app: kubernetes-nmstate (operand)
app: kubernetes-nmstate-operator (operator)
```
Since they serve to very different purpose, this way for example we can dump cluster wide logs with

```bash
# cluster wide operand logs
kubectl logs -n openshift-nmstate -l kubernetes-nmstate

# cluster wide operator logs
kubectl logs -n openshift-nmstate -l kubernetes-nmstate-operator
```